src/PublicStream.php

```diff
@@ -51,6 +51,16 @@ public function whenFrom(array $boundingBoxes, callable $whenFrom): self
 
     public function whenTweets(string | array $twitterUserIds, callable $whenTweets): self
     {
+        if (is_array($twitterUserIds) && count($twitterUserIds) > 1) {
+            array_reduce(
```

Comment:
This piece of code is unreadable to me. Could you make it more clear what this does, by performing the work in more steps and optionally extracting things to functions with a clear name?

Reply:
I'm not a PHP guy but as per my understanding, This piece of code concat all userIds from the array twitterUserIds into a string separated with commas.
For an example if twitterUserIds={1111, 2222, 3333, 4444} then it will return something "from:1111,2222,3333,4444".
I won't be able to modify the code as I am not an expert in PHP. So I would like to ask @anwsonwsymous to look into it if possible.
Anyway I have tested the changes and it works perfectly with multiple userIds otherwise it doesn't work (for me at-least).
I believe there are changes done in twitter API V2 stream endpoints and that broke the earlier code.

Reply:
I'm not a PHP guy So I would like to ask @anwsonwsymous to answer this if possible.

Reply:
The logic is
When we request Twitter api with this query **"from:111 from:222"** it will check that message received from both 111 and 222.
Instead we need to request this way **"from:111 OR from:222"** to receive all messages from 111 OR 222.
In other words, we need to add `OR` after each `from:` clause to make it work.

But here in this code I used `Laravel`'s alias function `last`,  instead we can use `php`'s `end` function to retrieve last element from ids.